rero_ils/modules/vendors/jsonschemas/vendors/vendor-v0.0.1.json

```diff
@@ -148,18 +211,32 @@
           "format": "email",
           "pattern": "^.*@.*\\..*$",
           "minLength": 6,
-          "validationMessage": "Email should have at least one <code>@</code> and one <code>.</code>"
+          "form": {
+            "validation": {
+              "messages": {
+                "pattern": "Email should have at least one <code>@</code> and one <code>.</code>"
+              }
+            }
+          }
         }
       }
     },
     "organisation": {
       "title": "Organisation",
       "type": "object",
+      "required": [
+        "$ref"
+      ],
       "properties": {
         "$ref": {
           "title": "Organisation URI",
           "type": "string",
-          "pattern": "^https://ils.rero.ch/api/organisations/.*?$"
+          "pattern": "^https://ils.rero.ch/api/organisations/.*?$",
+          "form": {
```

Comment:
this is useless as organisation is not displayed

Reply:
done